src/ActionList/Item.tsx

```diff
@@ -224,10 +230,13 @@ export function Item(itemProps: Partial<ItemProps> & {item?: ItemInput}): JSX.El
           <LeadingVisual />
         </LeadingVisualContainer>
       )}
-      <StyledTextContainer descriptionVariant={descriptionVariant}>
-        <div>{text}</div>
-        {description && <DescriptionContainer>{description}</DescriptionContainer>}
-      </StyledTextContainer>
+      {children}
```

Comment:
If we do retain `text` (https://github.com/primer/components/pull/1196/files#r624194497), should it be mutually-exclusive with `children`?